### PR TITLE
Feature/language icons

### LINKED ISSUE #30: Add icon for each language on cards
Icon for each language on cards

### PR DESCRIPTION
**Pull request by JeffShepherd**
## What does this PR do?

- Adds dynamic rendering of images to match the language displayed on each Card

## How should it be tested in the UI?

- Run ```npm start``` and ensure all cards display the relevant image next to the language

### CODE REVIEW

**Comment by JeffShepherd** [comment]
Styling in Cards will need to be perfected once main styling branch in process is merged in.

File: src/Components/Form/Form.js [hunk 1]
```diff
@@ -9,7 +9,6 @@ const Form = ({filterValue, sortByLanguage}) => {
         <select value={filterValue} onChange={(e) => sortByLanguage(e.target.value)} id="languageFilter">
           <option value=""></option>
           <option value="C">C</option>
-          <option value="C+">C+</option>
```
Comment (josharagon) [inline src/Components/Form/Form.js:12]: Nice catch

File: src/Components/Card/Card.js [hunk 2]
```diff
@@ -7,8 +7,13 @@ import Modal from '../Modal/Modal'
 
 
 const Card = ( {username, summary, language, date, id, addReview} ) => {
-
-  const path = `./assets/${language}.png`
+  let path;
+  
+  if(language === 'C#') {
```
Comment (josharagon) [inline src/Components/Card/Card.js:12]: Well done

**Review by josharagon — APPROVED** [review]
Nice touch to our app. After I merge with main, I will implement it inside of my branch with the necessary styling updates.

File: src/Components/Form/Form.js (hunk 1 above)
Comment (JeffShepherd) [reply src/Components/Form/Form.js:12]: Yea I realized C+ isn't an actual language when looking for a C+ icon :)

**Pull request merged**